srcpkgs/palapeli/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'palapeli'
+pkgname=palapeli
+version=18.08.3
+revision=1
+build_style=cmake
+hostmakedepends=extra-cmake-modules
+makedepends="libkdegames-devel kdoctools"
+depends=qhull
```

Comment:
quote

Reply:
Done. Quoted hostmakedepends as well on assumption that that was the preferred style.